lesson_07/homework_07.py

```diff
@@ -0,0 +1,175 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= 5:
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if  result > 25:
+            # Enter the action to take if the result is greater than 25
+            pass
```

Comment:
You need to take an action here. For ex. break or raise any exception. With pass keyword you only ignore this block of code and if check has no any sense.

Reply:
Thank you for your advice, I fixed it.